frame/system/src/lib.rs

```diff
@@ -1449,7 +1449,12 @@ impl<T: Config> Module<T> {
 			match r {
 				Ok(_) => Event::ExtrinsicSuccess(info),
 				Err(err) => {
-					sp_runtime::print(err);
+					log::trace!(
```

Comment:
can we make it debug?
```suggestion
					log::debug!(
```

Reply:
The point of this pr is to not to be use debug.

Reply:
It used to be unconditionally print, default log level is info. For failed extrinsic, I think it is fair to make it higher verbose level than trace?

Reply:
Right. I thought sp_runtime::print is info level (I think it used to be?).

Reply:
No. It was just `println!` back in the days, until we changed it to `log::debug!`